src/actions/search.js

```diff
@@ -57,6 +57,16 @@ export const searchAccount = ({ activePeer, address }) =>
     });
   };
 
+export const setSearchTranactions = searchObj => ({
```

Comment:
IMHO, this action is only used to set the filter, no need to pass address, transactions or count. Also the name of the action is not reflecting the change in the state.